introduction.md

```diff
@@ -0,0 +1 @@
+Hi! I'm Justin Lee. I'm a developer. I'm passionate about education, technology, video games, and all forms of design. Love HTML and CSS, brushing up on JavaScript. Happy coding!
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.